ecosystem/typescript/sdk/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 All notable changes to the Aptos Node SDK will be captured in this file. This changelog is written by hand for now. It adheres to the format set out by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+- Add support for Option<T> and vector<Object<T>> in the SDK
```

Comment:
I think is was released..? if so, let's add it under the release version section

Reply:
nvm, it is not released